src/spt_setup.c

```diff
@@ -369,8 +377,10 @@ get_argc_argv(int *argc_o, char ***argv_o)
     int rv = -1;
 
 #ifndef IS_PYPY
+#ifndef __APPLE__
```

Comment:
You haven't really explained the rationale of this change. What doesn't work? Why it works this way?